docs/kbs_attestation_protocol.md

```diff
@@ -438,92 +435,7 @@ eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.
 SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
 ```
 
-The next sections describe how the three strings that compose the token are
-formatted and built.
-
-##### Header
-
-This is the Base64url encoding of a [JOSE](https://jose.readthedocs.io/en/latest/)
-header.
-
-It is JSON-formatted and contains both the token standard format (`typ`) and the
-token signature algorithm (`alg`). For example:
-
-```json
-{
-    "typ": "JWT"
-    "alg": "RS256",
-}
-
-```
-
-##### Payload
-
-This is the Base64url encoding of a set of JSON-formatted claims, like the
-following example:
-
-``` json
-{
-    "exp": 1568187398,
-    "iat": 1568158598,
-    "iss": "https://xxxxxx",
-
-    "tee-pubkey": $pubkey
-}
-```
-
-The token payload is divided into registered and private claims sets. The KBS
-might also include additional public claims set.
-
-The registered claims set must include the `exp`, `iat`' and `iss` names, which
-respectively declare the expiration time, issuing time and issuer (KBS URL
-address) of the token:
-
-| Field | Description     | Type                    | Example              |
-|-      |-                |-                        |-                     |
-| `exp` | Expiration time | Seconds since the epoch | `99991231235959`     |
-| `iat` | Issuing time    | Seconds since the epoch | `180322235959`       |
-| `iss` | Issuer          | KBS URL                 | `https://my-kbs.io/` |
-
-The private claims set must include the `tee-pubkey` name. This name refers to
-the HW-TEE's public key sent by the KBC along with the attestation evidence,
-which is valid within the validity period of the token.
-
-When the KBC uses the token to request resources or services from a KBS service
-API, then the KBS uses this public key to encrypt the symmetric key used to
-encrypt the `output`.
-
-##### Signature
-
-Using the algorithm described in the token header, the KBS signs the
-concatenation of the token header and the token payload, separated by a dot
-(`.`). The token signature is the Base64url encoded string for that
-signature.
-
-#### HTTP Response
-
-The token is included in the `/kbs/v0/resource/token/` `GET` HTTP response
-body, as JSON content:
-
-``` json-with-comments
-{
-    "token": "$serialized_token"
-}
-```
-
-where `$serialized_token` is built as follows:
-
-``` rust
-let jwt_header = base64_url::encode(r#"{{"alg": "RS256","typ": "JWT"}}"#);
-
-let jwt_claims_string = format!(r#"{{"exp": {}, "iat": {}, "iss": {}, "tee-pubkey": {}}}"#, exp, iat, iss, pubkey);
-let jwt_claims = base64_url::encode(&jwt_claims_string);
-
-let jws_signature_input = format!("{}.{}", jwt_header, jwt_claims);
-let jws_signature = base64_url::encode(rs256_key_pair.sign(&jws_signature_input));
-
-let serialized_token = format!("{}.{}.{}", jwt_header, jwt_claims, jwt_signature);
-```
+Where the custom claims in the payload is the attestation result from the Attention Service.
```

Comment:
```suggestion
The `JWT` payload is the set of attestation result custom Claims, as generated by the Attestation Service.
```

Also, shouldn't we require for the generated JWT to include those registered claims:

``` json
{
    "exp": 1568187398,
    "iat": 1568158598,
    "iss": "https://xxxxxx",
    "tee-pubkey": $pubkey
}
```
```

Reply:
Yes, I agree. In the future, we can implement both the attestation result token and the pubkey registration token here, and I will keep the description in the document. Of course, this PR only implements the former.